README.md

```diff
@@ -73,7 +73,7 @@ Please see the compatibility matrix above before you deploy the driver
 
 To deploy the CSI driver:
 ```sh
-kubectl apply -k "https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/deploy/kubernetes/overlays/stable"
+kubectl apply -k "https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/deploy/kubernetes/overlays/stable/?ref=v0.1.0-alpha.2"
```

Comment:
This doesn't work as it already released and will pick older version image, we should rename this to `v0.1.0-alpha.3` and then cut the release immediately and then try pulling.

Reply:
updated.